modelon/impact/client/entities/custom_function.py

```diff
@@ -111,80 +116,90 @@ def parameter_values(self) -> Dict[str, Any]:
         """Custom_function parameters and value as a dictionary"""
         return {p.name: p.value for p in self._param_by_name.values()}
 
-    def get_compiler_options(self) -> ExecutionOptions:
+    def _get_options(self, use_defaults):
+        return (
+            self._custom_func_sal.custom_function_default_options_get(
+                self._workspace_id, self._name
+            )
+            if use_defaults
+            else self._custom_func_sal.custom_function_options_get(
+                self._workspace_id, self._name
+            )
+        )
```

Comment:
I think this expression is too long to be a one liner. For readability consider changing it to:
```
if use_defaults
  return self._custom_func_sal.custom_function_default_options_get(self._workspace_id, self._name)

return self._custom_func_sal.custom_function_options_get(self._workspace_id, self._name)
```